.github/workflows/lint-ts.yml

```diff
@@ -14,6 +14,10 @@ on:
             - benchmarks/utilities/*
             - .github/workflows/lint-ts.yml
 
+concurrency:
+    group: node-lint-${{ github.head_ref || github.ref }}
```

Comment:
this one is quick, no?  I'm not sure why we have a separate workflow for this, but do we really care? 
...I mean, it also doesn't hurt.